pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -162,6 +170,17 @@ var _ = Describe("Node-labeller ", func() {
 		))
 	})
 
+	It("should add cpu tsc labels if cpu counter exists and its name is tsc", func() {
```

Comment:
Please consider converting this test case into a table:
1. Test that no labels were added when there is no tsc counter.
2. Test the `v1.CPUTimerLabel+"tsc-scalable"` label when `Scaling` is true.
3. Same when its false.

Reply:
On a second thought, the first TC will not play nicely with the other two in a single table.

Reply:
In that case, I'll be proceeding with 2 and 3.

Reply:
You could have an "It()" for the first test case, and a table for the other two.

Reply:
Done.